src/js/InvertedIndex.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.files = [];
+  }
+
+  /**
+   * Validates file content
+   * @function
+   * @param {Array} fileContent content of uploaded file
+   * @return {Array|Boolean} valid fileContent or false if invalid file
+   */
+  static validateFile(fileContent) {
+    if (!(fileContent instanceof Array) || fileContent.length < 1) {
+      return false;
+    }
+
+    let invalid = false;
+    fileContent.forEach((book) => {
+      if (!(book.title && book.text)) {
+        invalid = true;
+        return;
+      }
+      if (book.text.length < 1) invalid = true;
+    });
+
+    if (invalid) return false;
+    return fileContent;
+  }
+
+  /**
+   * Strips out special characters and removes duplicate words
+   * @function
+   * @param {String} text text from the book
+   * @return {Array} An array of unique words in the text
+   */
+  static tokenize(text) {
+    return text.replace(/[^\w\s-]/g, '')
+      .toLowerCase().split(/\s+/)
+      .filter((word, index, collection) => collection.indexOf(word) === index);
+  }
```

Comment:
I don't think this code takes care of integers and white spaces, refactor your code to watch for them